packages/leancode_flutter_svg_adaptive_loader/README.md

```diff
@@ -0,0 +1,74 @@
+# leancode_flutter_svg_adaptive_loader
+
+[![leancode_flutter_svg_adaptive_loader pub.dev badge][pub-badge]][pub-badge-link]
+[![][build-badge]][build-badge-link]
+
+This package adds an adaptive bytes loader for `.svg` assets. That allows you to compile your vector assets by `vector_graphics_compiler`
+into binary either inside of your repository or on CI/CD, without thinking in-code whether the asset is xml-based or a compiled binary.
```

Comment:
I'd add a sentence or two explaining that it allows you to provide a path to a file that can be either a normal SVG or this precompiled one using `vector_graphics_compiler`. The "without thinking" part is a little bit vague imo

Reply:
Updated, please verify if I explained it less vaguely :D